src/routes/JobSummary/components/RecordsTable.js

```diff
@@ -117,142 +112,243 @@ export const RecordsTable = ({
-      const sourceRecord = jobLogRecords.find(item => item.sourceRecordId === sourceRecordId);
-      const entityId = sourceRecord?.relatedInstanceInfo.idList[0];
-      const path = `/inventory/view/${entityId}`;
-
-      const isPathCorrect = !!entityId;
-      const isHotlink = isPathCorrect && (instanceActionStatus === RECORD_ACTION_STATUS.CREATED
-        || instanceActionStatus === RECORD_ACTION_STATUS.UPDATED);
+    }) => (
+      <InstanceCell
+        instanceActionStatus={instanceActionStatus}
+        sourceRecordId={sourceRecordId}
+        jobLogRecords={jobLogRecords}
+      />
+    ),
+    holdingsStatus: ({ sourceRecordId }) => {
+      const {
+        instanceData,
+        holdingsData,
+        itemData,
+      } = getRelatedInfo(jobLogRecords, sourceRecordId);
+
+      if (isEmpty(holdingsData)) {
+        return (
+          <BaseLineCell>
+            <NoValue />
+          </BaseLineCell>
+        );
+      }
+
+      const itemInfo = itemData ? [...itemData] : [{}];
+      const instanceId = instanceData?.idList[0];
+
+      holdingsData?.forEach(holdings => {
+        const isDiscarded = holdings.actionStatus === RECORD_ACTION_STATUS.DISCARDED;
+        const holdingsId = holdings.id;
+
+        if (isDiscarded && !itemInfo.find(item => item.holdingsId === holdingsId)) {
+          itemInfo.push({ holdingsId, error: true });
+        }
+      });
 
-      return getHotlinkCellFormatter(isHotlink, entityLabel, path, 'instance');
+      return (
+        <HoldingsCell
+          instanceId={instanceId}
+          holdingsInfo={holdingsData}
+          itemInfo={itemInfo}
+          locations={locations.records}
+        />
+      );
     },
-    holdingsStatus: ({
-      holdingsActionStatus,
+    itemStatus: ({
       sourceRecordId,
+      itemActionStatus,
     }) => {
-      const entityLabel = getRecordActionStatusLabel(holdingsActionStatus);
-      const sourceRecord = jobLogRecords.find(item => item.sourceRecordId === sourceRecordId);
-      const instanceId = sourceRecord?.relatedInstanceInfo.idList[0];
-      const holdingsId = sourceRecord?.relatedHoldingsInfo.idList[0];
-      const path = `/inventory/view/${instanceId}/${holdingsId}`;
+      const {
+        instanceData,
+        holdingsData,
+        itemData,
+      } = getRelatedInfo(jobLogRecords, sourceRecordId);
+
+      const isGeneralItemError = isGeneralItemsError(itemData, itemActionStatus);
+
+      if (isGeneralItemError) {
+        return (
+          <BaseLineCell>
+            <FormattedMessage id="ui-data-import.logLight.actionStatus.noAction" />
+          </BaseLineCell>
+        );
+      }
+
+      if (isEmpty(itemData) && isEmpty(holdingsData)) {
+        return (
+          <BaseLineCell>
+            <NoValue />
+          </BaseLineCell>
+        );
+      }
+
+      if (isEmpty(itemData)) {
+        const emptyValues = holdingsData?.map((_, index) => (
+          <div key={index} style={{ paddingBottom: '7px' }}>
```

Comment:
Could you please investigate should we move to component `<div key={index} style={{ paddingBottom: '7px' }}>` or `paddingBottom: '7px' ` to css class and reuse across all pull request?